stdlib/int64.mli

```diff
@@ -114,6 +126,12 @@ external to_int : int64 -> int = "%int64_to_int"
    is taken modulo 2{^31}, i.e. the top 33 bits are lost
    during the conversion. *)
 
+val unsigned_to_int : int64 -> int
```

Comment:
Could you please document what happens if number can't be represented in as an unsigned number in the resulting `int`. Personally I would still argue that this should return an `int option`.

Reply:
Thanks for the keeping track of this! I changed the return type to `int option`, which I agree is better.